src/components/RangeInput.vue

```diff
@@ -80,14 +80,12 @@ export default {
     min: {
       type: Number,
       required: false,
-      // @major: remove this default
-      default: -Infinity,
```

Comment:
Not related to the changes itself, but do you know why we had this in the first place?

Reply:
Not sure, probably as a safety net? Anyway [in InstantSearch.js](https://github.com/algolia/instantsearch.js/blob/4bea07b99f492441eb94e483378e0778f90c5b43/src/connectors/range/connectRange.ts#L80:L89), we assume it's optional and take care of it.

Reply:
Yes, that's why, there were a couple places where we repeat the default values as InstantSearch as props, but this one couldn't be removed without breaking change

Reply:
Ok thanks!